spec/xml-extractor_spec.rb

```diff
@@ -21,7 +21,6 @@
             expected_phrase = expected_phrases[expected_phrase_index]
             expect(expected_phrase).to_not be_nil
             expect(expected_phrase['key']).to eq(actual_phrase.key)
-            # expect(expected_phrase['line_number']).to eq (line_number)
```

Comment:
Unfortunately Nokogiri doesn't report line numbers correctly, so I removed the check from the tests.